README.md

```diff
@@ -24,6 +26,25 @@ sudo apt install facerec
 source ~/.bashrc
 ```
 
+### Using .deb package
+
+#### 1. Download the latest release of Facerec from the releases section of the repository
+
+#### 2. Install the .deb package on your machine using the command
+```
+sudo dpkg -i <name of the .deb file>
+```
+
+### 3. Install All the dependencies
+```
+sudo apt install -y python3-pip python3-opencv python3-setuptools python-execnet cmake libatlas-base-dev build-essential
```

Comment:
put this as step 2 Because we require dependencies before installation of the deb package.